packages/Search/src/DTK_LinearBVH_decl.hpp

```diff
@@ -66,8 +65,37 @@ class BoundingVolumeHierarchy
   private:
     friend struct Details::TreeTraversal<DeviceType>;
 
-    Kokkos::View<Node *, DeviceType> _leaf_nodes;
-    Kokkos::View<Node *, DeviceType> _internal_nodes;
+    Kokkos::View<Node *, DeviceType> getInternalNodes()
+    { assert( !empty() ); return Kokkos::subview( _internal_and_leaf_nodes, std::make_pair(size_type{0}, size() - 1) ); }
+
+    Kokkos::View<Node *, DeviceType> getLeafNodes()
```

Comment:
Shouldn't it be `const`?

Reply:
You mean `View<Node *>` or mark the function member as `const`?

Reply:
The function

Reply:
It's only called from the constructor plus we return non-const view so no

Reply:
I just want some consistency. `getRoot()` returns non-const pointer to internal data, but is declared `const`. I don't see how this function is different.